libsplinter/src/biome/oauth/store/mod.rs

```diff
@@ -417,6 +417,10 @@ impl OAuthUserIter {
             inner: Box::new(users.into_iter()),
         }
     }
+
+    pub fn total(&self) -> usize {
```

Comment:
Perhaps this iterator should just implement `ExactSizeIterator`, like it's inner value.

Reply:
Updated this commit to implementing the `ExactSizeIterator` trait for the `OAuthUserIter` struct.